### PR TITLE
Update Documentation

### LINKED ISSUE #1501: Building Documentation
Several issues here when trying to build the documentation:

1. Should the [instructions](http://www.coolprop.org/develop/documentation.html) (minus step 0. for mac OS) work on Windows using Anaconda?  Has this been done?  I got it to work, sort of, but with much jumping through hoops and it doesn't all build correctly.  I had to fetch many 64-bit versions of the requisite modules and install the wheel files manually.  There's still some weird error at the beginning of the build about matplotlib calls being out of order.

2. There is a serious problem with converting the fluid_properties/mixtures.rst file.  This one is not just me.  Check out the [mixtures page](http://www.coolprop.org/dev/fluid_properties/Mixtures.html) on the CoolProp/dev documentation.  Something going wrong here in the ipython examples and this happens throughout the picture.  @ibell made a commit to this .rst file in the offending area some time back, but I don't think this is causing the errors.  ipython seems to think that binary pair data for 'Helium' and 'Xenon' already exist; yet when I run these scripts manually in Anaconda, they run fine.

3. Another error occurs that floods the stdout with messages and I can't capture where it is coming from.  It looks, though, like an example is trying to generate plots and it is calling flash routines with input parameters that are out of bounds.  See sample message below:

    c:\program files\anaconda2\lib\site-packages\CoolProp\Plots\Common.py:622: UserWarning: An error occurred for inputs 210905.712095, 3269.231633 with index (206L,): HmolarSmolar: [HSFLSH error 200] h-s inputs are two-phase or out of bounds, iterative routine is not available to find a solution.   

    You have to do a clean build (make clean html) to get these error message to print.

4. Lots of errors for missing .csv and tag files that it can't find as well as a couple of include files.

5. Oh yeah, and : **WARNING: logo file '_static/CoolPropLogo.png' does not exist**  This is easy to fix, but shouldn't it be in _static/  on the repository?

### PR DESCRIPTION
**Pull request by henningjp**
This is strictly a Documentation update (closes #1501 ).  Changes include:
- [x] Added documentation page for IF97 Backend under Fluid Properties doctree.
- [x] Updated HighLevelAPI doc page to include a section on calculating Vapor, Liquid, Saturation states (including latent heat calculation) that may head off some common new user questions.
- [x] Updated the instructions for building the documentation to include Windows Anaconda builds.
- [x] Minor additions to some of the .gitignore files because the doc build scripts don't clean up very well (at leas on the Windows Anaconda build).  Generally it leaves .rst.in files and jpeg and png files laying around after they are created by the scripts.

I'm completely open to changes/suggestions and can attach pdf's of the resulting pages if that would help, so you don't have to build them yourself.

### CODE REVIEW

**Comment by ibell** [comment]
That all looks pretty much good to me!

File: Web/develop/documentation.rst [hunk 1]
```diff
@@ -62,4 +67,120 @@ Building the documentation on Linux
...
+
+    pip install -vvv --pre
+     --trusted-host www.coolprop.dreamhosters.com
+     --find-links http://www.coolprop.dreamhosters.com/binaries/Python/ -U 
+     --force-reinstall CoolProp
+   
+
+**Run Setup Scripts**
+
+There are a number of setup scripts that have to be run to generate dynamic content for the web documentation.  To run them, open a windows command prompt (cmd) and cd to the CoolProp\\Web\\scripts directory.  Then run the following scripts::
```
Comment (jowr) [inline Web/develop/documentation.rst]: You might want to refer to the script runner at https://github.com/CoolProp/CoolProp/blob/master/Web/scripts/__init__.py#L107

**Comment by henningjp** [comment]
On hold until issue #1548 is resolved.  It's needed to view the Mixtures page properly.

**Comment by henningjp** [comment]
@jowr added reference to __init__.py

Also added more commits to:
 - [x] Fix Mixtures page following incorporation of #1562 
 - [x] Print commands in several docs now work under both Python 2 & 3
 - [x] Updates to docs to compile Python wrapper

This PR is dependent on PR #1562

**Comment by henningjp** [comment]
Don't merge.  This is too badly out of sync with other merges.  I'll create a new, cleaner, PR.

**Comment by ibell** [comment]
Good, I was going to ask about this PR

**Comment by henningjp** [comment]
Closing this PR as it got too far behind some big changes on master.  New clean PR coming as a replacement.

**Pull request closed without merging**